### ISSUE TITLE
Build errors

### ISSUE THREAD

**Issue by katyhuff** [kamuda1/dissertation#7]
There are a bunch of pdflatex (or latex) build errors in this repo if it's cloned clean. 
Please confirm that everything is being committed appropriately. Please consider adding a makefile.

Lots of errors like this one:

```
! Package pdftex.def Error: File `images/cnn_kernel_length.png' not found: usin
g draft setting.

See the pdftex.def package documentation for explanation.
Type  H <return>  for immediate help.
 ...

l.411 ...\textwidth]{images/cnn_kernel_length.png}
```
Also, there is some kind of unicode issue in the bibfile.

```

LaTeX Warning: Citation `CASANOVAS2012588' on page vii undefined on input line
13.


! Package inputenc Error: Unicode character \GenericError {(inputenc)
      }{Package inputenc Error: Unicode character × (U+D7)
(inputenc) not set up for use with LaTeX}{See the inputenc package documentatio
n for explanation.}{You may provide a definition with
(inputenc) \DeclareUnicodeCharacter } (U+D7)
(inputenc) not set up for use with LaTeX.

See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...

l.14 ...rom \cite {MOSZYNSKI2006739}.\relax }}{18}
```

**Comment by katyhuff** [comment]
I have merged #8 which solved most issues, but I am still getting the following unicode errors with a clean clone and pdflatex.


```
! Package inputenc Error: Unicode character \GenericError {(inputenc)
      }{Package inputenc Error: Unicode character × (U+D7)
(inputenc) not set up for use with LaTeX}{See the inputenc package documentatio
n for explanation.}{You may provide a definition with
(inputenc) \DeclareUnicodeCharacter } (U+D7)
(inputenc) not set up for use with LaTeX.

See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...

l.14 ...rom \cite {MOSZYNSKI2006739}.\relax }}{18}
```

```

[17 <./images/USGS_k_conc.png>]

! Package inputenc Error: Unicode character × (U+D7)
(inputenc)                not set up for use with LaTeX.

See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...

l.180 ...Reproduced from \cite{MOSZYNSKI2006739}.}

?
````

**Comment by kamuda1** [comment]
Issue closed with #11. Main issues addressed were images referenced in the document not included in the images folder and the `\bibliography{refs}` command being called incorrectly as `\bibliography{refs.bib}`. The `\bibliography{refs.bib}` command searches for the file `refs.bib.bib`, which does not exist.
